### ISSUE TITLE
ClassCastException 2.4-beta1

### ISSUE THREAD

**Issue by SlimeDog** [SkriptLang/Skript#2267]
### Steps to Reproduce

- update to Skript 2.4-beta1
- start server
- observe console/log

### Errors / Screenshots

```
[03:33:13] [Server thread/ERROR]: #!#!
[03:33:13] [Server thread/ERROR]: #!#! [Skript] Severe Error:
[03:33:13] [Server thread/ERROR]: #!#!
[03:33:13] [Server thread/ERROR]: #!#! Something went horribly wrong with Skript.
[03:33:13] [Server thread/ERROR]: #!#! This issue is NOT your fault! You probably can't fix it yourself, either.
[03:33:13] [Server thread/ERROR]: #!#! You should report it at https://github.com/SkriptLang/Skript/issues. Please copy paste this report there (or use paste service).
[03:33:13] [Server thread/ERROR]: #!#! This ensures that your issue is noticed and will be fixed as soon as possible.
[03:33:13] [Server thread/ERROR]: #!#!
[03:33:13] [Server thread/ERROR]: #!#! Stack trace:
[03:33:13] [Server thread/ERROR]: #!#! java.lang.ClassCastException: class java.lang.String cannot be cast to class org.bukkit.Location (java.lang.String is in module java.base of loader 'bootstrap'; org.bukkit.Location is in unnamed module of loader 'app')
[03:33:13] [Server thread/ERROR]: #!#!     at ch.njol.skript.lang.function.FunctionReference.execute(FunctionReference.java:210)
[03:33:13] [Server thread/ERROR]: #!#!     at ch.njol.skript.lang.function.ExprFunctionCall.get(ExprFunctionCall.java:44)
[03:33:13] [Server thread/ERROR]: #!#!     at ch.njol.skript.lang.util.SimpleExpression.getArray(SimpleExpression.java:102)
[03:33:13] [Server thread/ERROR]: #!#!     at ch.njol.skript.effects.EffBroadcast.execute(EffBroadcast.java:66)
[03:33:13] [Server thread/ERROR]: #!#!     at ch.njol.skript.lang.Effect.run(Effect.java:52)
[03:33:13] [Server thread/ERROR]: #!#!     at ch.njol.skript.lang.TriggerItem.walk(TriggerItem.java:61)
[03:33:13] [Server thread/ERROR]: #!#!     at ch.njol.skript.lang.TriggerItem.walk(TriggerItem.java:89)
[03:33:13] [Server thread/ERROR]: #!#!     at ch.njol.skript.lang.Trigger.execute(Trigger.java:57)
[03:33:13] [Server thread/ERROR]: #!#!     at ch.njol.skript.events.EvtScript.register(EvtScript.java:61)
[03:33:13] [Server thread/ERROR]: #!#!     at ch.njol.skript.ScriptLoader$3.call(ScriptLoader.java:754)
[03:33:13] [Server thread/ERROR]: #!#!     at ch.njol.skript.ScriptLoader$3.call(ScriptLoader.java:1)
[03:33:13] [Server thread/ERROR]: #!#!     at ch.njol.skript.ScriptLoader.loadScript(ScriptLoader.java:773)
[03:33:13] [Server thread/ERROR]: #!#!     at ch.njol.skript.ScriptLoader.lambda$1(ScriptLoader.java:406)
[03:33:13] [Server thread/ERROR]: #!#!     at ch.njol.skript.ScriptLoader.loadScripts(ScriptLoader.java:425)
[03:33:13] [Server thread/ERROR]: #!#!     at ch.njol.skript.ScriptLoader.lambda$0(ScriptLoader.java:347)
[03:33:13] [Server thread/ERROR]: #!#!     at ch.njol.skript.ScriptLoader.loadScripts(ScriptLoader.java:377)
[03:33:13] [Server thread/ERROR]: #!#!     at ch.njol.skript.Skript$2.run(Skript.java:534)
[03:33:13] [Server thread/ERROR]: #!#!     at org.bukkit.craftbukkit.v1_14_R1.scheduler.CraftTask.run(CraftTask.java:81)
[03:33:13] [Server thread/ERROR]: #!#!     at org.bukkit.craftbukkit.v1_14_R1.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:394)
[03:33:13] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_14_R1.MinecraftServer.b(MinecraftServer.java:1023)
[03:33:13] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_14_R1.DedicatedServer.b(DedicatedServer.java:396)
[03:33:13] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_14_R1.MinecraftServer.a(MinecraftServer.java:971)
[03:33:13] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_14_R1.MinecraftServer.run(MinecraftServer.java:816)
[03:33:13] [Server thread/ERROR]: #!#!     at java.base/java.lang.Thread.run(Thread.java:834)
[03:33:13] [Server thread/ERROR]: #!#!
[03:33:13] [Server thread/ERROR]: #!#! Version Information:
[03:33:13] [Server thread/ERROR]: #!#!   Skript: 2.4-beta1
[03:33:13] [Server thread/ERROR]: #!#!     Flavor: skriptlang-github
[03:33:13] [Server thread/ERROR]: #!#!     Date: 2019-07-20
[03:33:13] [Server thread/ERROR]: #!#!   Bukkit: 1.14.4-R0.1-SNAPSHOT
[03:33:13] [Server thread/ERROR]: #!#!   Minecraft: 1.14.4
[03:33:13] [Server thread/ERROR]: #!#!   Java: 11.0.3 (Java HotSpot(TM) 64-Bit Server VM 11.0.3+12-LTS)
[03:33:13] [Server thread/ERROR]: #!#!   OS: Linux amd64 3.10.0-957.21.3.el7.x86_64
[03:33:13] [Server thread/ERROR]: #!#!
[03:33:13] [Server thread/ERROR]: #!#! Server platform: Spigot
[03:33:13] [Server thread/ERROR]: #!#!
[03:33:13] [Server thread/ERROR]: #!#! Current node: null
[03:33:13] [Server thread/ERROR]: #!#! Current item: broadcast testing("test")
[03:33:13] [Server thread/ERROR]: #!#! Current trigger: script load (script load) (function.sk, line 11)
[03:33:13] [Server thread/ERROR]: #!#!
[03:33:13] [Server thread/ERROR]: #!#! Thread: Server thread
[03:33:13] [Server thread/ERROR]: #!#!
[03:33:13] [Server thread/ERROR]: #!#! Language: english
[03:33:13] [Server thread/ERROR]: #!#! Link parse mode: DISABLED
[03:33:13] [Server thread/ERROR]: #!#!
[03:33:13] [Server thread/ERROR]: #!#! End of Error.
[03:33:13] [Server thread/ERROR]: #!#!
```

### Server Information
* **Server version/platform:**       Spigot 1.13.2 final or Spigot 1.14.4 on CentOS/Linux (up-to-date)
* **Skript version:**                2.4-beta1

**Comment by bensku** [comment]
I think I know what this is. Will fix later today.

**Comment by Matocolotoe** [comment]
Tested it with the change on the pull request, no more errors.

**Comment by Blueyescat** [comment]
bensku knows how to add a condition and your code is wrong

**Comment by Matocolotoe** [comment]
Sorry, just wanted to help...

**Comment by FranKusmiruk** [comment]
@Blueyescat please be polite in the issue tracker. No one is going to be shot even if the code was wrong, we appreciate the intention.

**Comment by SlimeDog** [comment]
Confirmed fixed in Skript 2.4-beta2. Thanks.
